lib/juicebox.ex

```diff
@@ -1,5 +1,4 @@
 defmodule Juicebox do
-  use Application
```

Comment:
You should keep it in, it ensures your application first the application contract, it also allows you to stop the application. https://github.com/elixir-lang/elixir/blob/v1.3.3/lib/elixir/lib/application.ex#L149